thirdparty/github.com/go-yaml/OWNERS

```diff
@@ -0,0 +1,28 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
if these packages are not going to be referenced outside this library, they should be under an `internal` subpackage

if they are going to be referenced outside this module, I wouldn't expect callers to have to use import paths containing thirdparty and irrelevant github.com subpackage names

I was imagining something more like sigs.k8s.io/yaml/goyaml.v2 or something similar

Reply:
> if these packages are not going to be referenced outside this library, they should be under an internal subpackage

We (kustomize) need them to be available outside this library. I will rename their import paths to `sigs.k8s.io/yaml/goyaml.v2` and `sigs.k8s.io/yaml/goyaml.v3`